### ISSUE TITLE
Packaged app cannot be installed if uap5:DriverDependency section in appxmanifest.xml is set without any content

### ISSUE THREAD

**Issue by petercoin** [microsoft/WindowsAppSDK#4123]
Hi,
I'd like to know is it an expected behavior that the installation will be abandoned if the uap5:DriverDependency section is declared in the appxmanifest file without any valid content like
```
    <uap5:DriverDependency>
      <!--<uap5:DriverConstraint Name="xxx_hsa.inf" MinVersion="1.1.1.1"/>-->
    </uap5:DriverDependency>
```
My application is packaged by Windows Application Packaging project. The installation process will be stopped with error code: 0x80073CFD and it will show the device drivers are missing but it doesn't show any driver information. I think this is caused by the empty driver information in XML above section.

**Comment by DrusTheAxe** [comment]
>expected behavior that the installation will be abandoned if the uap5:DriverDependency section is declared in the appxmanifest file without any valid content

Confirmed. The manifest schema allows `<uap5:DriverDependency>` to have no child elements but the deployment pipeline processing this expects at least 1. So don't specify DriverDependency if you have no DriverConstraint(s). In your example broaden the comment

```
    <!--
    <uap5:DriverDependency>
      <uap5:DriverConstraint Name="xxx_hsa.inf" MinVersion="1.1.1.1"/>
    </uap5:DriverDependency>
    -->
```

Thanks for reporting this. I've opened a bug as we shouldn't fail here but it's relatively low priority given how easily it's worked around. Plus it's easily detected by any testing so it shouldn't get past development into the wild.
